datahub-graphql-core/src/main/resources/timeline.graphql

```diff
@@ -0,0 +1,189 @@
+extend type Query {
+    """
+    Get list of schema blame
+    """
+    getSchemaBlame(input: GetSchemaBlameInput!): GetSchemaBlameResult
+}
+
+type TimelineConfig {
+    """
+    Whether the timeline feature is enabled.
+    """
+    enabled: Boolean!
+}
+
+"""
+Enum of SemanticVersionChange
+"""
+enum SemanticVersionChangeType {
+    """
+    No semantic version change
+    """
+    NONE
+    """
+    Patch semantic version change
+    """
+    PATCH
+    """
+    Major semantic version change
+    """
+    MAJOR
+    """
+    Minor semantic version change
+    """
+    MINOR
+    """
+    Exceptional semantic version change
+    """
+    EXCEPTIONAL
+}
+
+"""
+Enum of types of changes
+"""
+enum ChangeOperationType {
+    """
+    When an element is added
+    """
+    ADD
+    """
+    When an element is modified
+    """
+    MODIFY
+    """
+    When an element is removed
+    """
+    REMOVE
+}
+
+"""
+Enum of CategoryTypes
+"""
+enum ChangeCategoryType {
+    """
+    When documentation has been edited
+    """
+    DOCUMENTATION
+    """
+    When glossary terms have been added or removed
+    """
+    GLOSSARY_TERM
+    """
+    When ownership has been modified
+    """
+    OWNERSHIP
+    """
+    When technical schemas have been added or removed
+    """
+    TECHNICAL_SCHEMA
+    """
+    When tags have been added or removed
+    """
+    TAG
+}
+
+"""
+Input for getting schema changes computed at a specific version.
+"""
+input GetSchemaBlameInput {
+    """
+    The dataset urn
+    """
+    datasetUrn: String!
+    """
+    Categories to filter on
+    """
+    categories: [ChangeCategoryType!]
+    """
+    Changes after this version are not shown
+    """
+    version: String
+}
+
+"""
+Schema changes computed at a specific version.
+"""
+type GetSchemaBlameResult {
+    """
+    Latest and current semantic version
+    """
+    latestVersion: SemanticVersionStruct!
+    """
+    Selected semantic version
+    """
+    version: SemanticVersionStruct
+    """
+    All semantic versions
+    """
+    semanticVersionList: [SemanticVersionStruct!]
+    """
+    List of schema blame
+    """
+    schemaFieldBlameList: [SchemaFieldBlame!]
```

Comment:
Should this array be required? if so add a ! after the end

Reply:
It doesn't have to be.

Reply:
What does it mean if its not there?

Reply:
It is absent when we query for just the versions

Reply:
Addressed in comments.